src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -176,7 +176,8 @@
 
     <ItemGroup>
       <FunctionalTest>
-        <Command>$(HelixPythonPath) $(RunnerScript) --dll %(Filename).dll $(OtherRunnerScriptArgs) -- $(XunitArgs)</Command>
+        <Command Condition="'$(TargetsWindows)' == 'true'">$(HelixPythonPath) $(RunnerScript) --script Runtests.cmd %HELIX_CORRELATION_PAYLOAD% %HELIX_WORKITEM_ROOT%\execution</Command>
```

Comment:
`%HELIX_WORKITEM_ROOT%\execution` is that actually needed? Will Runtests.cmd already be in that directory? If so I wouldn't even pass this parameter and let it default the same as we do locally. Just another thing that might get slightly out of sync what we don't need.